src/zenml/models/v2/core/service_account.py

```diff
@@ -51,6 +52,10 @@ class ServiceAccountRequest(BaseRequest):
         title="A description of the service account.",
         max_length=TEXT_FIELD_MAX_LENGTH,
     )
+    external_user_id: Optional[UUID] = Field(
```

Comment:
This should probably not be something that regular users can set in this request (or the update model below)?